<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedStore</class>
 <widget class="QWidget" name="FeedStore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="feedstore_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(56, 59, 67);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="feedstore_min_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="feedstore_max_restore_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="feedstore_cose_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(56, 59, 67);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(68, 71, 77);</string>
        </property>
        <widget class="NavButton" name="feedstore_detail_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>饲料药品仓库明细</string>
         </property>
        </widget>
        <widget class="NavButton" name="feedstore_incom_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>入库</string>
         </property>
        </widget>
        <widget class="NavButton" name="feedstore_out_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>出库</string>
         </property>
        </widget>
        <widget class="NavButton" name="feedstore_return_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>退货</string>
         </property>
        </widget>
        <widget class="NavButton" name="area_detail_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>分仓明细</string>
         </property>
        </widget>
        <widget class="NavButton" name="area_return_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>分仓退库</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="feedstore_stackedWidget">
        <property name="minimumSize">
         <size>
          <width>768</width>
          <height>581</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(68, 71, 77);</string>
        </property>
        <widget class="QWidget" name="page_6">
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>170</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>400</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page">
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>20</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>190</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavButton</class>
   <extends>QPushButton</extends>
   <header location="global">qnavbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
